pandas/core/generic.py

```diff
@@ -4717,24 +4717,41 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         applied as a function even if callable. Used in __setitem__.
         """
         inplace = validate_bool_kwarg(inplace, 'inplace')
+        inplace = True if inplace else False
```

Comment:
no this just makes things confusing, do this below in a condiationl, don't modify ``inplace``